python/orion_finance_sdk_py/asset_map.py

```
@@ -0,0 +1,46 @@
+    from .contracts import OrionConfig
+
+    config = OrionConfig()
+    address_map: dict[str, str] = {}
+
+    for asset in config.whitelisted_assets:
+        testnet = Web3.to_checksum_address(asset)
+        try:
+            twin = config.w3.eth.contract(address=testnet, abi=_MAINNET_SOURCE_ABI)
+            mainnet = twin.functions.mainnetSource().call()
+        except Exception:
+            continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)